api/lib/round-tracker.js

```diff
@@ -329,17 +329,46 @@ async function defineTasksForRound (pgClient, sparkRoundNumber, taskCount) {
   ])
 }
 
+// Exponentially look at more blocks to handle the case when we have an outage
+// and the rounds are not advanced frequently enough, while keeping the happy
+// path performant.
+export async function getRoundStartEpochWithBackoff (
+  contract,
+  roundIndex,
+  maxAttempts = 5
+) {
+  for (let attempt = 0; attempt < maxAttempts; attempt++) {
+    try {
+      return await getRoundStartEpoch(
+        contract,
+        roundIndex,
+        50 * Math.pow(2, attempt)
```

Comment:
Nitpick, feel free to ignore:

```suggestion
        50 * 2 ** attempt
```

Or alternatively to make it clear what is the precedence of operators:

```suggestion
        50 * (2 ** attempt)
```